.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,3 @@
 ## Description
-<!--
-Please add an informative description that covers the changes made by the pull request.
 
-If applicable, reference the bug/issue that this pull request fixes here.
--->
-
-This checklist is used to make sure that common guidelines for a pull request are followed.
-
-- [ ] **I have read the [contribution guidelines](./CONTRIBUTING.md).**
```

Comment:
Instead of removing the entire thing, how about just removing the redundant ones. Lets continue keeping the following:
 [ ] **I have read the [contribution guidelines](./CONTRIBUTING.md).** 
 [ ] Title of the pull request is clear and informative. 
 [ ] Pull request includes test coverage for the included changes.

The main reason is that I want people to put conscious effort into adding tests. This might work as a reminder to them.